### DISCUSSION TITLE
What are the Installation Questions

### DISCUSSION THREAD

**Discussion by orionpilot** [retorquere/zotero-better-bibtex#2484] [Q&A] [answered]
I just installed Zotero and BetterBibTex. In the installation instructions, it says: "On first install you will be asked a few questions. Please read these instead of just clicking through." However, after restarting zotero, I never got those questions. What were they? I am pretty sure they will help me set up the plugin correctly so I'd like to know.

**Comment by retorquere** [comment]
If first-run doesn't show that means BBT must have been installed at some point in the profile. You can restart the first-run process:

* go into the [hidden preferences](https://www.zotero.org/support/preferences/hidden_preferences)
* find `citekeyFormat`
* right-click it and choose `reset`
* restart Zotero

**Comment by orionpilot** [comment]
That makes sense, as I have used Zotero + BBT many years ago. I installed them again to see how things have changed. 

Following your instructions, I was able to reset the `citekeyFormat` preference. However, the `citekeyFormatEditing` preference does not want to be reset (see image). I mention that since after following these steps and restarting Zotero, I still don't get the first-run questions.
 
![image](https://user-images.githubusercontent.com/32915653/231486007-79000d3c-ee03-45bd-a79b-d3ca0a7b1395.png)

However, if the first-run just asks what format I want the citekey, then I know how to change that. I just want to make sure that's the only thing.

**Comment by retorquere** [comment, marked as answer]
It asks for the format, whether you want to restart so drag and drop works immediately. It used to ask more, but that just defaults to `true` now, so the firstrun dialog isn't as important as it used to be.

**Comment by orionpilot** [comment]
Sounds good, thanks for the information!
